includes/publisher/lib/field-functions.php

```diff
@@ -183,6 +183,9 @@ function append_reverse_relationship_fields( array $models, string $post_type ):
 function sanitize_field( string $type, $value ) {
 	switch ( $type ) {
 		case 'text':
+			if ( is_array( $value ) ) {
+				return $value;
```

Comment:
Probably worth sanitizing the values here so they're treated the same way as the single value:

```suggestion
				return array_map( 'wp_strip_all_tags', $value );
```